controllers/flowlogspipeline/flp_common_objects.go

```diff
@@ -576,3 +578,47 @@ func (b *builder) serviceMonitor() *monitoringv1.ServiceMonitor {
 	}
 	return &flpServiceMonitorObject
 }
+
+func (b *builder) prometheusRule() *monitoringv1.PrometheusRule {
+	flpPrometheusRuleObject := monitoringv1.PrometheusRule{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      b.prometheusRuleName(),
+			Labels:    b.labels,
+			Namespace: b.namespace,
+		},
+		Spec: monitoringv1.PrometheusRuleSpec{
+			Groups: []monitoringv1.RuleGroup{
+				{
+					Name: "NetobservFlowLogsPipeline",
+					Rules: []monitoringv1.Rule{
+						{
+							Alert: "NetobservNoFlows",
+							Annotations: map[string]string{
+								"description": "Netobserv flowlogs-pipeline is not receiving any flow, this is either a connection issue with the agent, or an agent issue",
```

Comment:
Can you replace `Netobserv` with `NetObserv` (capital O) ? There's the occurrence in several places in this file